### ISSUE TITLE
Admin Page: encoding issues in translated strings

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#5088]
#### Steps to reproduce the issue

1. Go to Settings > General
2. Switch your site language to French
3. Update to Jetpack 4.3

![crrimjtwyaagttc](https://cloud.githubusercontent.com/assets/426388/18275432/3d9c11ee-7447-11e6-9fe6-bf04f789d28b.jpg)

**Comment by oskosk** [comment]
This may be happening because React sanitizes strings before rendering. This includes HTMLencoding some characters. It may be also happening in plain English too, depending on where are we getting those strings from.

**Comment by oskosk** [comment]
@jeherve In this particular case from the screenshot, look's like it's because we're using `’` in the String. Do you think that this character needs to be kept in French, or a regular `'` is appropriate too?

**Comment by jeherve** [comment]
The character needs to be kept in French. `'` is not an apostrophe in French.
